src/game/counting.rs

```diff
@@ -0,0 +1,276 @@
+use crate::game::cards::Card;
```

Comment:
i don't think it will do more.  i separated it because of two reasons: first, "counting" is a distinct phase in the game and it just "feels" like the phase of the game is a stronger affinity than scoring.  this is easy to argue the other way. Second, all the tests are in the module and the tests are very different.